src/app/components/dynamic-list/DynamicList.jsx

```diff
@@ -25,12 +26,16 @@ const DynamicList = props => {
                 })}
             </ul>
         );
+    } else if (props.loading) {
+        list = <div className="loader loader--dark" />;
     } else {
         list = <div aria-live="assertive">{props.noResultsText}</div>;
     }
     return (
         <div>
-            <HeadingLevel className="dynamic-list__title">{props.title}</HeadingLevel>
+            <HeadingLevel className="dynamic-list__title" data-testid="dynamic-list-title">
```

Comment:
are you using this `data-testid="dynamic-list-title"` ? I haven't seen test for this component, did you forget to push it?

Reply:
Ah yes, the test uses the className 'dynamic-list__title' but should instead use this new ID